src/vmm/src/guest_config/x86_64/cpuid/amd/normalize.rs

```diff
@@ -292,18 +292,19 @@ impl super::AmdCpuid {
     ) -> Result<(), ExtendedCacheTopologyError> {
         for i in 0.. {
             if let Some(subleaf) = self.get_mut(&CpuidKey::subleaf(0x8000001d, i)) {
-                // Cache type. Identifies the type of cache.
+                // Cache level. Identifies the level of this cache. Note that the enumeration value
+                // is not necessarily equal to the cache level.
```

Comment:
nit: could include the following in this PR:
```diff
-                        set_range(&mut subleaf.result.eax, 14..28, sub)
+                        set_range(&mut subleaf.result.eax, 14..26, sub)
```

Reply:
Added fix for this as additional commit.